subsys/bluetooth/shell/gatt.c

```diff
@@ -108,13 +111,25 @@ static int cmd_exchange_mtu(const struct shell *sh,
 		return -ENOEXEC;
 	}
 
+	if (exchange_params.func) {
+		shell_print(sh, "Shell command busy. A previous invocation is in progress.");
+		return -EAGAIN;
+	}
+	exchange_params.func = exchange_func;
+
 	err = bt_gatt_exchange_mtu(default_conn, &exchange_params);
+
+	if (err) {
+		/* Release global `exchange_params`. */
+		exchange_params = (struct bt_gatt_exchange_params){};
+	}
+
 	if (err == -EALREADY) {
 		shell_print(sh, "Already exchanged");
 	} else if (err) {
 		shell_print(sh, "Exchange failed (err %d)", err);
 	} else {
-		shell_print(sh, "Exchange pending");
+		shell_print(sh, "Ok. Exchange pending");
```

Comment:
```suggestion
		shell_print(sh, "Exchange pending");
```

The "Ok." doesn't really add anything useful.